sklearn/utils/tests/test_extmath.py

```diff
@@ -489,7 +515,7 @@ def one_pass_var(X):
     def two_pass_var(X):
         mean = X.mean(axis=0)
         Y = X.copy()
-        return np.mean((Y - mean)**2, axis=0)
+        return np.mean((Y - mean) ** 2, axis=0)
```

Comment:
This one is not a PEP8 so mostly down to personal preferences. In general, I would avoid changing things like this who are not related to the PR goals.